src/styles/global.css

```diff
@@ -52,6 +52,24 @@ body {
 
 .prose :is(h1, h2, h3, h4, h5, h6) a {
   text-decoration: none !important;
+  background-image: linear-gradient(45deg, #151f38, #7a7ebd);
+  color: transparent;
+  background-clip: text;
+  -webkit-background-clip: text;
+}
+
+.prose :is(h1, h2, h3, h4, h5, h6) a:hover {
+  text-decoration: underline !important;
+  transition:
+    color 0.2s ease-in-out,
+    background-color 0.2s ease-in-out,
+    text-decoration-thickness 0.2s;
+  color: #151f38;
+}
+
+.prose :is(h1, h2, h3, h4, h5, h6) a:focus {
+  outline: 4px solid #0073e6;
+  outline-offset: 5px;
```

Comment:
looks funny, but I like it! can we update the `--color-primary-active` as well? `#3c3ffa` is from the 2023 design system. 

![image](https://github.com/user-attachments/assets/700b54c6-44ea-492c-9229-056da072a703)

Reply:
the outline color `#3c3ffa` doesn't look nice in this design system, though - is it possible to change it to `#151f38`? it can be done in a separate PR later.